backend/tests/test_production_evidence_v9.py

```
@@ -0,0 +1,175 @@
+            "strategy_stages": {"hype": True}, "quality_gates": {"cross_exchange_confirmed": False},
+            "ticker": {"last": 1.0, "mark": 1.0, "vwap": 1.1},
+            "liquidation_flow": {"available": True, "observed_at": 109, "long_liquidation_notional_1m": 100000.0, "short_liquidation_notional_1m": 1000.0, "liquidation_velocity_usd_per_min": 100000.0, "burst_ratio": 2.0},
+            "cascade_intelligence": {"status":"PASS","readiness_points":7.0,"maximum_available":10.0,"components":{"liquidations":{"available":True,"points":1.2,"maximum":2.0}}},
+        },
+    }
+
+
+def test_v9_records_replay_context_and_exact_liquidation_without_mutating_input(tmp_path):
+    db_path = str(migrate_test_database(tmp_path / "evidence.db"))
+    recorder = ProductionEvidenceRecorder(db_path)
+    result = _result(); before = copy.deepcopy(result)
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)